homeassistant/components/victron_gx_mqtt/__init__.py

```diff
@@ -0,0 +1,68 @@
+"""The victron_mqtt integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from homeassistant.const import Platform
+
+if TYPE_CHECKING:
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import HomeAssistant
+
+from .hub import Hub
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [
+    Platform.SENSOR,
+]
+
+type VictronGxConfigEntry = ConfigEntry[Hub]
+
+
+async def _update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Handle options update."""
+    _LOGGER.info("Options for victron_mqtt have been updated - applying changes")
+    # Reload the integration to apply changes
+    await hass.config_entries.async_reload(entry.entry_id)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: VictronGxConfigEntry) -> bool:
+    """Set up victronvenus from a config entry."""
+    _LOGGER.debug("async_setup_entry called for entry: %s", entry.entry_id)
+
+    hub = Hub(hass, entry)
+    entry.runtime_data = hub
+
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
```

Comment:
We should by the way only load the platforms when we can confirm we can connect to the device